src/Sentry.Unity/SentryUnity.cs

```diff
@@ -85,4 +85,10 @@ public static CrashedLastRun GetLastRunState()
 
         return UnitySdk.CrashedLastRun();
     }
+
+    /// <summary>
+    /// Captures a User Feedback
+    /// </summary>
+    public static void CaptureFeedback(string message, string? email, string? name, bool addScreenshot) =>
```

Comment:
Hm this is odd that we add an API here because it creates further confusion between `SentrySdk` vs `SentryUnity`. Both will have `CaptureFeedback` from the `SentrySdk` and get the screenshot taken before calling it and doing `s => s.AddAttachment`?

Reply:
also not totally against adding this here, just sayijng it's not ideal

Reply:
> I think the .NET SDK one should be hidden along with its Init.


How do you capture events? It'll be on `SentrySdk`, right? some stuff on one, some on another